financial-connections/src/main/java/com/stripe/android/financialconnections/features/notice/NoticeSheet.kt

```diff
@@ -74,7 +74,8 @@ private fun NoticeSheetContent(
             is Generic -> GenericBottomSheetContent(
                 screen = content.generic,
                 onClickableTextClick = onClickableTextClick,
-                onConfirmModalClick = onConfirmModalClick
+                onPrimaryButtonClick = onConfirmModalClick,
+                onSecondaryButtonClick = {} // TODO handle secondary button clicks.
```

Comment:
Still unused on generic sheets, exposed it specifically for the update required modal but I'll look into improving this in a more generic way soon.